### ISSUE TITLE
PROBLEM

### ISSUE THREAD

**Issue by ftila** [stanislav-web/tecdoc#1]
добрый вечер,
Я недавно скачал проект TECDoc но, к сожалению, я не смог его включить меня есть проблема с переменной $APPLET ($APPLICATION)
Вы можете помочь мне !!!

**Comment by stanislav-web** [comment]
Да, $Applet это идентификатор приложения на котором работает TecDoc. ТО есть это ваша CMS. По умолчанию там Bitrix

**Comment by klikict** [comment]
And how do we integrate it? I have same error but use Prestashop and Woocommerce..
